src/ingest.py

```diff
@@ -48,8 +53,12 @@ def ingest(event, config, ntl_data_formatter, socrata_data_formatter, elasticsea
         slack_notifier.sendSlackNotification("Error ingesting " +
                                              event['datasource'] + " ==> " + str(e))
 
-
 def makeQueryCall(queryURL):
     r = requests.get(queryURL)
     response = json.loads(r.text)
     return response
+
+if (__name__ == '__main__'):
+    event={'datasource': str(sys.argv[1])}
```

Comment:
Please document what you pass in here and perhaps validate the argument to prevent usage errors. Could add instructions to a readme or an example run script, or perhaps use argparse's ability to give help messages on invalid params